geoportal/c2cgeoportal_geoportal/views/layers.py

```diff
@@ -195,7 +195,7 @@ def _proto_read(self, layer: "main.Layer") -> FeatureCollection:
 
     @view_config(route_name="layers_read_many", renderer="geojson")  # type: ignore
     def read_many(self) -> FeatureCollection:
-        set_common_headers(self.request, "layers", Cache.NO)
+        set_common_headers(self.request, "layers", Cache.PRIVATE_NO)
```

Comment:
Be careful here as we are in editing module, I would have recommended no-store for such views.
But it seems we can use private with 10 seconds as, after save ngeo seems to load altered feature directly from PUT response.